### PR TITLE
isolate rendering of markdown into a single React component

### PR DESCRIPTION
**Pull request by Hamms**
A component that currently uses `marked` for markdown rendering, but will eventually be updated to use remark.

### CODE REVIEW

**Comment by Hamms** [comment]
[<CdoMarkdown content=”...”/> React component for consistent client-side rendering of CDO-flavored markdown](https://trello.com/c/DopuuG51/237-cdomarkdown-content-react-component-for-consistent-client-side-rendering-of-cdo-flavored-markdown)

File: apps/src/templates/RenderedMarkdown.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * new remark system, and possibly even support redaction.
+ */
+export default class RenderedMarkdown extends React.Component {
+  static propTypes = {
+    markdown: PropTypes.string.isRequired
+  };
+
+  render() {
+    const processedMarkdown = processMarkdown(this.props.markdown, { renderer });
+    /* eslint-disable react/no-danger */
```
Comment (joshlory) [inline apps/src/templates/RenderedMarkdown.jsx]: I worry a little bit about hiding this warning inside a slick wrapper — consumers may not realize the `markdown` prop is vulnerable to unsafe HTML.  I wonder if we should call this `UnsafeRenderMarkdown` for clarity, until we start limiting what markup can be rendered?

**Comment by joshlory** [comment]
Love the approach!  One question about sharp edges / foot-guns.

File: apps/src/templates/RenderedMarkdown.jsx (hunk 1 above)
Comment (Hamms) [reply apps/src/templates/RenderedMarkdown.jsx]: Ooooh, I like this.

**Pull request merged**